ntrans_gui.py

```diff
@@ -0,0 +1,260 @@
+import webbrowser
+import tkinter
+import functools
+from tkinter import (
+    filedialog,
+)  # TODO: Figure out why I can't remove this import and use tkinter.filedialog... "self.savepath = tkinter.filedialog.askdirectory()" gives AttributeError: module 'tkinter' has no attribute 'filedialog'
```

Comment:
This is a  weird python feature. But if you `import tkinter.filedialog`, then it works like you wanted.

Reply:
Well, my goal was to delete one row of imports, so this becomes the same. But yeah, still prefer "tkinter.filedialog"